### DISCUSSION TITLE
HM-Sec-RHS Farbe der Icons ändert sich nicht mit dem Status

### DISCUSSION THREAD

**Discussion by Telebear** [SukramJ/aiohomematic#1386] [Q&A]
Beim HM-Sec-SCo ändert sich die Farbe des Icons in den Entitäten-/ Übersichts-Karten. Bei den Fensterdrehgriffen HM-Sec-RHS leider nicht. Dies erschwert die Erkennbarkeit des Status des Fensters.
Kann ich das irgendwie anpassen? Leider finde ich keinen Hinweis darauf.
Der HM-Sec-SCo wird als "binary_sensor.<name>", der HM-Sec-RHS als "sensor.<name>" angegeben.
![Unbenannt](https://github.com/danielperna84/hahomematic/assets/13784573/a659a102-4585-4d59-9a96-9e640e201696)

**Comment by SukramJ** [comment]
Ich denke es macht Sinn die [Dublette](https://homematic-forum.de/forum/viewtopic.php?f=84&t=81431&p=793557&sid=b24cef80538cf1bfff99a111684cf98b#p793557) hier zu schliessen.
